core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -1658,11 +1658,14 @@ boolean isTxConsistent(final Transaction tx, final boolean isSpent) {
         boolean isActuallySpent = true;
         for (TransactionOutput o : tx.getOutputs()) {
             if (o.isAvailableForSpending()) {
```

Comment:
the following change is not intuitively correct. Is `o.getSpentBy() != null` such a common case that it matters enough to change it?

Reply:
I suppose that the order of two if checks changes nothing? 
But I've decided to follow method docu and stop method execution if even a one output is available. So you're right could just write return true at the end of this if branch, would do.